### PR TITLE
Added install script 

### PR DESCRIPTION
**Pull request by lcskrishna**
Added an installation script.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+apt-get install -y sqlite3 libsqlite3-dev libfmt-dev
```
Comment (mwootton) [inline install.sh:1]:
```suggestion
#!/bin/bash
apt-get install -y sqlite3 libsqlite3-dev libfmt-dev
```

Comment (mwootton) [reply install.sh:1]: You may need to check the permissions on the script.  It should be 755 when you pull.
Thanks.

**Review by mwootton — COMMENTED** [review]
This is a great idea, thanks.

File: install.sh (hunk 1 above)
Comment (lcskrishna) [reply install.sh:2]: Done

**Comment by lcskrishna** [comment]
@mwootton updated to use with 755 when cloned and addressed the comment.

**Pull request merged**